components/metrics/metrics_log.cc

```diff
@@ -265,12 +268,14 @@ const SystemProfileProto& MetricsLog::RecordEnvironment(
   if (client_->GetBrand(&brand_code))
     system_profile->set_brand_code(brand_code);
 
+#if !defined(STARBOARD)
```

Comment:
Likewise as above, we have Starboard CPUInfo API -> i'd add a follow-up bug to look at plumbing that data in through CPUInfo.

Reply:
Added a TODO